daskhub/Chart.yaml

```diff
@@ -1,12 +1,12 @@
 apiVersion: v2
 name: daskhub
-icon: https://avatars3.githubusercontent.com/u/17131925?v=3&s=200
+icon: https://avatars3.githubusercontent.com/u/1.1.2925?v=3&s=200
```

Comment:
```suggestion
icon: https://avatars3.githubusercontent.com/u/17131925?v=3&s=200
```

Reply:
The replace operation interpreted `.` as the regexp `.` which all characters accepted, and then matched on 17131 as something to be replaced as well ;D